.ci/scripts/distribution/qa-testbench.sh

```diff
@@ -1,7 +1,50 @@
-#!/bin/sh -eux
+#!/bin/bash -eux
 
 chmod +x clients/go/cmd/zbctl/dist/zbctl
 
-alias zbctl="clients/go/cmd/zbctl/dist/zbctl"
+zbctl="clients/go/cmd/zbctl/dist/zbctl"
 
-zbctl create instance qa-protocol --variables "${QA_RUN_VARIABLES}"
+"${zbctl}" create instance external-tool-integration --variables "${QA_RUN_VARIABLES}"
+
+businessKey="${BUSINESS_KEY}"
+
+waiting=1
+
+echo "Waiting for result of $businessKey"
+
+while [ $waiting -eq 1 ]; do
```

Comment:
You can do

```suggestion
while true; do
```

And use a `break` statement to break out of the loop instead of setting the variable to 1. So for example:

```bash


while true ; do
    ...
    if [ -z "$key" ]; then
        ...
    else
        echo "QA run completed"
        break
    fi
done
```